packages/oui-message/src/index.spec.js

```diff
@@ -1,153 +1,150 @@
-            expect(getTextMessageFromTransclusion(textTranscludeElement)).toBe(message);
-        });
+            const message = "My message";
+            const element = TestUtils.compileTemplate(`<oui-message>${message}</oui-message>`);
 
-        it("should display text message from binding", () => {
-            expect(getTextMessageFromBinding(textBindingElement)).toBe(message);
+            expect(getBody(element)).toBe(message);
         });
 
-        it("should display success icon", () => {
-            expect(getIcons(textBindingElement, "oui-icon-success_circle").length).toBe(1);
-        });
+        describe("when info type", () => {
+            it("should display info icon", () => {
+                const element = TestUtils.compileTemplate("<oui-message type=\"info\"></oui-message>");
 
-        it("should display close button", () => {
-            expect(getCloseButton(textBindingElement)).toBeDefined();
-        });
+                expect(getIcons(element, "oui-icon-info_circle").length).toBe(1);
+            });
 
-        it("should close message when close button is clicked", () => {
-            getCloseButton(textBindingElement).click();
+            it("should display close button", () => {
+                const element = TestUtils.compileTemplate("<oui-message type=\"info\"></oui-message>");
 
-            expect(messageExist(textBindingElement)).toBe(false);
-        });
-    });
+                expect(getCloseButton(element)).toBeDefined();
+            });
 
-    describe("When error type, it", () => {
-        beforeEach(() => {
-            textTranscludeElement = compileDirective(`<oui-message type="error">${message}</oui-message>`);
-            textBindingElement = compileDirective(`<oui-message type="error" message="${message}"></oui-message>`);
-        });
+            it("should dismiss message when close button is clicked", () => {
+                const element = TestUtils.compileTemplate("<oui-message type=\"info\"></oui-message>");
 
-        it("should display text message from transclusion", () => {
-            expect(getTextMessageFromTransclusion(textTranscludeElement)).toBe(message);
-        });
+                getCloseButton(element).click();
 
-        it("should display text message from binding", () => {
-            expect(getTextMessageFromBinding(textBindingElement)).toBe(message);
-        });
+                expect(isMessageDisplayed(element)).toBe(false);
+            });
 
-        it("should display error icon", () => {
-            expect(getIcons(textBindingElement, "oui-icon-error_circle").length).toBe(1);
-        });
+            it("should assign binding value to aria-label attribute on close button", () => {
+                const closeButtonLabel = "My label";
+                const element = TestUtils.compileTemplate(`<oui-message type=\"info\" aria-close-button-label="${closeButtonLabel}"></oui-message>`);
```

Comment:
No need to escape double-quotes since you are inside backticks.

Reply:
done